backend/src/services/__tests__/CMSAuthService.unit.test.ts

```diff
@@ -1,5 +1,59 @@
 import { MongoClient } from "mongodb";
+import CMSAuthService from "../CMSAuthService";
+import jwt from "jsonwebtoken";
 
-describe("hi", () => {
+/**
+ * @author ale8k
+ */
+describe("CMSAuthService", () => {
+    let connection;
+    let testJwt: string;
 
+    // Setup the DB for testing
+    beforeAll(async () => {
+        connection = await MongoClient.connect((global.__MONGO_URI__),{
+                poolSize: 10,
+                useNewUrlParser: true,
+                useUnifiedTopology: true
+            },
+        );
+        CMSAuthService.db = await connection.db(global.__MONGO_DB_NAME__);
+
+        await CMSAuthService.db.collection("cms-users").insertOne(
+            {
+                username: "alex",
+                password: "$2b$12$Cp/IwyOUsKiJZTANWklBB.k4mv07lIV1gSLT4FbtsOI.eoFi2qfTu"
+            }
+        );
+
+        testJwt = jwt.sign(
+            { username: "alex ", userId: "1" },
+            process.env.JWT_KEY as string,
+            { expiresIn: "2 days" }
+        );
+    });
```

Comment:
these credentials needs to be changed. Create a test on top level to test whether with a given credentials the user is connected or associated with the db.

Reply:
Not sure what you mean here? These are credentials being placed into the test db to be used to login lol.

Reply:
yeah.. if I have have a user with alex.. then these tests are doomed. I just wanted to say I can test my db with any credentials.

Reply:
They reset every suite man